src/d3a/models/market/blockchain_interface.py

```diff
@@ -101,8 +101,13 @@ def track_trade_event(self, trade):
             'trade_id': trade.id,
             'buyer': ALICE_STASH_ADDRESS,
             'seller': BOB_STASH_ADDRESS,
-            'amount': default_amount,
-            'rate': default_rate
+            'amount': {
+                'free': trade.offer.energy * BC_NUM_FACTOR,
+                'reserved': 0,
+                'miscFrozen': 0,
+                'feeFrozen': 0
+            },
```

Comment:
```suggestion
           'energy': trade.offer.energy,
```
Once this PR(https://github.com/gridsingularity/canvas-node/pull/7) from canvas-node side is merged

Reply:
Already had the commit lined up :)